### ISSUE TITLE
Can't close out polygon when you choose draw tool

### ISSUE THREAD

**Issue by ryanbranciforte** [trailheadlabs/outerspatial-map-library#49]
1. activate draw tool
2. choose area
3. click around to create polygon
4. try to complete polygon, it won't let you.

![screen shot 2017-09-25 at 5 56 32 pm](https://user-images.githubusercontent.com/2602563/30837611-ec22b3c4-a21a-11e7-880b-f15608a58849.png)

**Comment by nateirwin** [comment]
How are you trying to complete it? I believe, by default, the measure and draw tools "close" polygons by clicking on the first vertice in the polygon.

**Comment by ryanbranciforte** [comment]
Double clicking, which is how it works with measure/line

**Comment by nateirwin** [comment]
Ah yes, measure does support double-click in addition to clicking on the last vertice. I'll update the title to reflect that we should make this behavior consistent.

**Comment by nateirwin** [comment]
BTW, I can't reproduce being unable to close out a polygon with the measure tool.

**Comment by ryanbranciforte** [comment]
Same issue with measure tool, can't close out polygon on double click.

**Comment by nateirwin** [comment]
Oh, I see. I had some old code cached.

So... just to be clear: The behavior is consistent between the measure and draw tools. You can click on the first vertice to close a polygon (which is how we built it), but you want to be able to close the polygon by double-clicking (like ArcMap). Correct?

Sorry about all the back-and-forth on this. I should have verified I had the latest.

**Comment by ryanbranciforte** [comment]
Yes, that is correct. Double click is what I intuitively tried to do, so it feels like others will do the same. 

Also, a tool tip, maybe when you hover over the tools would be helpful, that way we could provide instructions. The double click would be less critical if it was clear how to close out a poly.

**Comment by nateirwin** [comment]
It's the ArcGIS in you :-) We'll get double-click added. We can do tooltips too, but that's probably going to take a little while.

**Comment by ryanbranciforte** [comment]
Sounds good. Actually, I think it's a right click/complete poly in Arc, but yeah, it's in me and I tried that as well :)

**Comment by ryanjhodge** [comment]
Could you guys test to see if you can finish the polygon/line features by double clicking in both draw and measure tools across browsers? Leaflet draw comes with this functionality out of the box but it it's getting disabled in chrome on desktop. This is because leaflet >1.0.0 detects the chrome browser as a touch enabled (even if you don't have a touchscreen) and in turn, leaflet turns off double click events on touch "devices"

**Comment by ryanbranciforte** [comment]
@ryanjhodge I tested on chrome and safari. The polys close on double click in safari for both measure and draw, but not in Chrome. 

If it is a hassle, not sure it's worth it to focus on this right now, might want to table this until later and move on the remaining 7.0.2 issues. Check in with @nateirwin on priorities. 

If we can eventually get tool tips in there that will help.

**Comment by nateirwin** [comment]
I'll take this one on.

Not sure we want to add tooltips. Ideally the symbols used for each button are intuitive enough on their own.

**Comment by ryanbranciforte** [comment]
Cool, I think the symbols are intuitive, but not the action needed to complete a poly.
